### DISCUSSION TITLE
All Features List

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#128] [Features: Implemented]
Hey @h3rald !New incomers, like me, may want to see a list of all discussions that was implemented and not implemented and yet implemented which are as options, in v0.31.0. So you should make an all list which contains all.

**Comment by h3rald** [comment]
Well... I do write a changelog containing all changes ever time a new version is released:

https://github.com/h3rald/min/releases

**Reply by ghost** [reply]
I know you write changelong on every updating. But it looks ... not enough to see what was/is/will be implemented, i think.

**Reply by h3rald** [reply]
Uhm I guess not. I do update the documentation though, so maybe I can include some excerpt from the doc in the change log.

Also, 0.31.0 was quite a distruttive update (I am still having trouble migrating *my own code*!) -- other releases hopefully shouldn't be that big.

**Reply by ghost** [reply]
> other releases hopefully shouldn't be that big.

There are so many talked features. i'm excited about future releases. 🤯

**Reply by ghost** [reply]
Thanks for that you declared feature states as categories, of the discussions 📁

**Reply by h3rald** [reply]
> Thanks for that you declared feature states as categories, of the discussions 📁

You're welcome. I had to find a way to keep things organized! Too confusing otherwise 😊
